Patch-Source: https://github.com/AlexanderThaller/hstdb/pull/56
--
From 958fa44a77846142374f53e2beb1da8c667fc637 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sun, 12 Mar 2023 19:55:13 +0100
Subject: [PATCH] Rename option --text_excluded to --text-excluded for
 consistency

All other options use kebab-case. The old name will still work, I've
specified it as an alias.
---
 src/opt.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/opt.rs b/src/opt.rs
index af7fb97..db9f6e1 100644
--- a/src/opt.rs
+++ b/src/opt.rs
@@ -193,7 +193,7 @@ struct DefaultArgs {
     command_text: Option<Regex>,
 
     /// Only print entries not containing the given regex
-    #[clap(short = 'T', long = "text_excluded")]
+    #[clap(short = 'T', long = "text-excluded", alias = "text_excluded")]
     command_text_excluded: Option<Regex>,
 
     /// Only print entries that have been executed in the current directory
